hudi-spark/src/main/java/org/apache/hudi/keygen/ComplexKeyGenerator.java

```diff
@@ -98,12 +94,4 @@ public HoodieKey getKey(GenericRecord record) {
 
     return new HoodieKey(recordKey.toString(), partitionPath.toString());
   }
-
-  public List<String> getRecordKeyFields() {
```

Comment:
Unused methods? Some annotation of the PR to reason about the changes would be great

Reply:
I've removed and refactored the KeyGenerator back to how it was before.